analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -66,7 +66,12 @@ import okhttp3.Request
 const val PIP_VERSION = "18.0"
 
 const val PIPDEPTREE_VERSION = "0.13.0"
-val PIPDEPTREE_DEPENDENCIES = arrayOf("pipdeptree", "setuptools", "wheel")
+val IGNORED_DEPENDENCIES = mapOf(
+    "pipdeptree" to "", // pipdeptree
+    "pkg-resources" to "0.0.0", // Added by a bug with some Ubuntu distributions
```

Comment:
Nit: Please add a dot at the end of the sentence.

Reply:
done.